README.md

```diff
@@ -94,6 +94,483 @@ Try the workshop at https://apprunnerworkshop.com
 Read the docs at https://docs.aws.amazon.com/apprunner
 ```
 
+## Use Pulumi resources with CDK Constructs
+
+It is possible to use Pulumi and CDK resources side-by-side. In order to pass a
+Pulumi Output value into a CDK resource you can use the [asString][./api-docs/README.md#asString], [asList][./api-docs/README.md#asList], &
+[asNumber](./api-docs/README.md#asNumber) functions. Conversely, in order to pass a CDK attribute to a Pulumi
+resource, you can use the [Stack.asOutput](./api-docs/README.md#asOutput) function to convert the CDK resource
+to a Pulumi Output value.
+
+### Example
+
+```ts
+import * as pulumicdk from '@pulumi/cdk';
+import * as aws from '@pulumi/aws';
+import * as s3ObjectLambda from 'aws-cdk-lib/aws-s3objectlambda';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+
+const app = new pulumicdk.App('app', (scope: pulumicdk.App) => {
+    const stack = new pulumicdk.Stack('accesspoint-stack');
+    const bucket = new s3.Bucket(stack, 'example-bucket');
+
+    const policyDoc = new iam.PolicyDocument();
+    policyDoc.addStatements(...);
+
+    const ap = new aws.s3.AccessPoint('exampleBucketAP', {
+      // Use `asOutput` to convert the bucketName attribute to a Pulumi Output
+      bucket: stack.asOutput(bucket.bucketName),
+      name: S3_ACCESS_POINT_NAME,
+      policy: policyDoc.toJSON(),
+    }, { parent: scope });
+
+    const objectLambdaAP = new s3ObjectLambda.CfnAccessPoint(stack, 's3ObjectLambdaAP', {
+      name: OBJECT_LAMBDA_ACCESS_POINT_NAME,
+        objectLambdaConfiguration: {
+          // Use `asString` to convert a Pulumi Output to a string value
+          supportingAccessPoint: pulumicdk.asString(ap.arn),
+          transformationConfigurations: [...],
+        },
+    });
+});
+```
+
+### L2 Example
```

Comment:
What does L2 exactly stand for in this case?
It seems like the test is importing a CDK resource created by the pulumi provider, what is L2 here?